packages/babel-helpers/src/helpers.js

```diff
@@ -2015,6 +2015,24 @@ helpers.classPrivateMethodGet = helper("7.1.6")`
   }
 `;
 
+helpers.classPrivateFieldInitSpec = helper("7.14.1")`
+  export default function _classPrivateFieldInitSpec(obj, privateMap, value) {
+    if (privateMap.has(obj)) {
+      throw new TypeError("redeclared private field identifier");
```

Comment:
We could copy the Firefox error message:
```suggestion
      throw new TypeError("Initializing an object twice is an error with private fields");
```

The problem here is technically not that the field is _declared_ twice (that would be a parsing error), but that it runs through the class initialization process twice.